cloudslang-compiler/src/main/java/io/cloudslang/lang/compiler/SlangSource.java

```diff
@@ -38,19 +40,33 @@ public static SlangSource fromFile(File file) {
 
         String source;
         try {
-            source = FileUtils.readFileToString(file, Charset.defaultCharset());
+            source = readFileToString(file);
         } catch (IOException e) {
             throw new RuntimeException("There was a problem reading the yaml file: " + file.getName(), e);
         }
         return new SlangSource(source, file.getName());
     }
 
+    public static String readFileToString(File file) throws IOException {
```

Comment:
or another approach is to use SlangSource also for loading inputs files (all kinds of files) - we are already doing it in the builder for `.input.yaml` files.. so if you use SlangSource in CompilerHelperImpl.loadMapsFromFiles method we will all be aligned to SlangSource and it will be easier to update in the future -- inthat case the moethods can stay here